mongo.js

```diff
@@ -253,10 +261,37 @@ LiveDbMongo.prototype._query = function(mongo, cName, query, fields, callback) {
     delete query.$count;
     mongo.collection(cName).count(query.$query || {}, function(err, count) {
       if (err) return callback(err);
-
       // This API is kind of awful. FIXME in livedb.
       callback(err, {results:[], extra:count});
     });
+
+  } else if (query.$distinct) {
+    delete query.$distinct;
+    mongo.collection(cName).distinct(query.$field, query.$query || {}, function(err, result) {
```

Comment:
@Dignifiedquire I am a bit worried here - distinct have limitation 16Mb and we had this limit on production.
but looks we do not use this functionality